indico/modules/rb_new/client/js/modules/admin/SettingsPage.jsx

```diff
@@ -244,6 +244,20 @@ const SettingsPage = props => {
                                 </Translate>
                             </p>
                         </Field>
+                        <Field name="grace_period"
+                               component={ReduxFormField}
+                               as="input"
+                               type="number"
+                               max="24"
+                               parse={value => (value ? +value : null)}
```

Comment:
This will also convert 0 to null. Also, the description below should mention the special meaning of leaving it empty.

Reply:
Ah, right. It is not needed now since we will allow empty values now.